README.md

```diff
@@ -8,12 +8,12 @@
 
```

Comment:
should we change the 
`[GitHub Workflow Status (branch)](https://img.shields.io/github/workflow/status/SymphonyPlatformSolutions/symphony-api-client-python/build/2.0)`
above /